dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -165,48 +167,46 @@ def generate_workshop_daily_session_summary(workshop)
     summary = {
       this_workshop: {},
     }
-    summary['questions'] = get_questions_for_forms
 
-    summary[:this_workshop] = generate_workshops_survey_summary(workshop)
+    questions = get_questions_for_forms(workshop)
+
+    summary['questions'] = questions
+
+    summary[:this_workshop] = generate_workshops_survey_summary(workshop, questions)
 
     summary[:facilitators] = Hash[*workshop.facilitators.pluck(:id, :name).flatten]
 
     summary
   end
 
-  def generate_workshops_survey_summary(workshop)
+  def generate_workshops_survey_summary(workshop, questions)
     surveys = get_surveys_for_workshops(workshop)
 
-    surveys['Pre Workshop'] = get_pre_workshop_surveys(workshop)
-    surveys['Post Workshop'] = get_post_workshop_surveys(workshop)
-
     workshop_summary = {}
 
-    get_questions_for_forms.each do |session, response_sections|
+    questions.each do |session, response_sections|
```

Comment:
What are `response_sections`? Please add a comment somewhere describing the data format. I'm having a hard time following this and keeping the terminology straight.